packages/react-components/source/scss/library/components/_content.scss

```diff
@@ -185,7 +185,7 @@
     background-color: $puppet-n100;
     border-radius: 2px;
     font-family: $puppet-type-font-family-monospace;
-    padding: $puppet-common-spacing-base / 2 $puppet-common-spacing-base;
+    padding: $puppet-common-spacing-base * 0.5 $puppet-common-spacing-base;
```

Comment:
Is it possible to use 
```
@use "sass:math";
.
.
.
padding: math.div($puppet-common-spacing-base, 2) $puppet-common-spacing-base;
```
instead?

Reply:
The [sass-migrator](https://sass-lang.com/documentation/cli/migrator) defaults to the `* 0.5` version. Either is fine with me.